tests/testsuite/check_cfg.rs

```diff
@@ -256,6 +269,7 @@ fn features_test() {
         .run();
 }
 
+#[allow(deprecated)]
 #[cargo_test]
 fn features_doctest() {
```

Comment:
I think the `does_not_contains` could be removed.  If we were passing `-Z`, it would fail as this test runs on stable